website/docs/r/vapp_org_network.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_org_network"
+sidebar_current: "docs-vcd-resource-vapp-org-network"
+description: |-
+  Provides capability to attach existing Org VDC Network to vApp and toggle network features.
+---
+
+# vcd\_vapp\_org\_network
+
+ Provides capability to attach existing Org VDC Network to vApp and toggle network features.
+
+Supported in provider *v2.7+*
+
+## Example Usage
+
+```hcl
+resource "vcd_vapp_org_network" "vappOrgNet" {
+  org = "my-org" #Optional
+  vdc = "my-vdc" #Optional
+
+  vapp_name         = "my-vapp"
+  org_network_name  = "my-org-network"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when 
+  connected as sysadmin working across different organisations.
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level.
+* `vapp_name` - (Required) The vApp this VM should belong to.
+* `org_network_name` - (Required) An Org network name to which vApp network is connected to.
+* `is_fenced` (Optional) Fencing allows identical virtual machines in different vApp networks connect to organization VDC networks that are accessed in this vApp. Default is false.
+* `firewall_enabled` - (Optional) Firewall service enabled or disabled. Configurable when `is_fenced` is true. Default is true. 
+* `nat_enabled` - (Optional) NAT service enabled or disabled. Configurable when `is_fenced` is true. Default is true.
```

Comment:
Probably worth noting that `nat_enabled` won't work if `firewall_enabled=false`

Reply:
changed